S23/vivekkatari/Main.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a word: ");
+        String word = scanner.nextLine();
```

Comment:
It would be easy to write a word in a new line and  it can be easy to find a word .

Reply:
Thanks for the suggestion, but i feel there wont be difference if we write code in different lines